pkg/api/componentreadiness/component_report.go

```diff
@@ -1118,3 +1122,11 @@ func (c *ComponentReportGenerator) getUniqueJUnitColumnValuesLast60Days(ctx cont
 func init() {
 	componentAndCapabilityGetter = testToComponentAndCapability
 }
+
+type TestVariants struct {
```

Comment:
Why did this one land here rather than in apis/api/crtest ?

Reply:
Its sole usage is in this file, `GenerateVariants` feeds it in as part of the cache key; it doesn't go out over the API at all, so I didn't think it made sense to mix it in. If it were to go anywhere there, I'd probably put it in `reqopts`? But now I'm thinking it would make sense to put it right next to `GenerateVariants` as it does sort of stick out here at the bottom.